misc/vim/syntax/nit.vim

```diff
@@ -86,9 +86,9 @@ syn match  NITSharpBang	"\%^#!.*"
 syn match  NITComment	"#.*" contains=NITTodo,@Spell
 
 " Keywords
-syn keyword NITKeyword	 abstract intern new
```

Comment:
Did you grep for the `'intern'` keyword in the SableCC grammar? I found no occurrence of it.